src/ape/cli/options.py

```diff
@@ -599,12 +599,27 @@ def config_override_option(**kwargs):
     return _json_option("--config-override", help="Config override mappings", **kwargs)
 
 
+def _excluded_compilers_callback(ctx, param, value):
+    if not value:
+        return
+
+    return [c.lower() for c in value]
+
+
 def excluded_compilers_option(**kwargs):
+    from ape.utils.basemodel import ManagerAccessMixin
+
+    registered_compilers_options = [
+        compiler.name
+        for compiler in ManagerAccessMixin.compiler_manager.registered_compilers.values()
+    ]
```

Comment:
They are already listed by name
```suggestion
    registered_compilers_options = list(
        ManagerAccessMixin.compiler_manager.registered_compilers
    )
```

Reply:
Now they are listed by file extension. Now to exclude a compiler, you should use the extension [.vy|.vyi|.sol|.json]. Isn't it more intuitive using  the compiler name: Vyper, Solidity?

Reply:
You are totally right, my apologies on that one